### PR TITLE
Display warning if user doesn't have rights to change permissions #4474

### PR DESCRIPTION
**Pull request by ghost**
Display warning if user doesn't have rights to change permissions #4474

### CODE REVIEW

File: tests/configure-test-env.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ python_exists=$(command -v python)
 if [ -z $python_exists ] 
 then 
   python_exists=$(command -v python3)
-  if [ -z python_exists]
+  if [ -z python_exists ]
```
Comment (allenmanning) [inline tests/configure-test-env.sh:20]: @cderv Can you please review?  This got my tests running without errors.

File: src/publish/confluence/confluence-verify.ts [hunk 2]
```diff
@@ -58,3 +59,25 @@ export const verifyConfluenceParent = async (
...
+) => {
+  const canManagePermissions = await client.canSetPermissions(
+    parent,
+    space,
+    user
+  );
+
+  if (!canManagePermissions) {
+    const confirmed: boolean = await Confirm.prompt(
+      "Published pages will be directly editable within the Confluence web editor.\n\nThis means that if you republish the page from Quarto, you may be overwriting the web edits. We recommend that you establish a clear policy about how this published page will be revised.\n\nAre you sure you want to continue?"
```
Comment (allenmanning) [inline src/publish/confluence/confluence-verify.ts:77]: @cwickham Can you please review this warning prompt message?

File: tests/configure-test-env.sh [hunk 3]
```diff
@@ -53,7 +53,7 @@ fi
 
 # Update tinytex
 echo ">>>> Configuring TinyTeX environment"
-if [ -z $GH_TOKEN ] && [ -n $(command -v gh) ]
+if [ -z $GH_TOKEN ] && [[ -n $(command -v gh) ]]
```
Comment (cderv) [inline tests/configure-test-env.sh:56]: I think the safest way if to write this way 
```suggestion
if [[ -z $GH_TOKEN && -n $(command -v gh) ]]
```

So many versions that I was not sure which one to use : http://mywiki.wooledge.org/BashFAQ/031

Hopefully this woks on Mac

**Review by cderv — APPROVED** [review]
Yes that is ok for this update on the sh script. If it works on your side with Mac, should be ok.

File: src/publish/confluence/confluence-verify.ts (hunk 2 above)
Comment (allenmanning) [reply src/publish/confluence/confluence-verify.ts:77]: That is a great suggestion, thank you.

File: tests/configure-test-env.sh (hunk 3 above)
Comment (allenmanning) [reply tests/configure-test-env.sh:56]: I will give that a shot.

**Pull request merged**